### ISSUE TITLE
Edit ReadME for accurate installation instructions

### ISSUE THREAD

**Issue by Houdanait** [gentzkow/template#36]
Hello @gentzkow, 
The step (2) of the README to install `conda` and `oracle_jdk` says : 
```
brew cask install miniconda
brew cask install oracle-jdk
```

This turns out to be an erroneous syntax, and should be : 

```
brew install --cask miniconda
brew install --cask oracle-jdk
```

If you're ok, i am going to amend the README for correct instructions. 

EDIT: I am not able to assign myself, and probably don't have admin on this repo to make ammendments on branch/merge.

**Comment by gentzkow** [comment]
Yes. Thanks! Looks like the other Homebrew syntax was [recently deprecated](https://github.com/ansible-collections/community.general/issues/1524).

Remember to make yourself the assignee.

**Comment by Houdanait** [comment]
@gentzkow Matt, I am not able to make myself assignee on any of the issues related to your template (cf comment [above](https://github.com/gentzkow/template/issues/36#issue-845253511)) there is no option for me to assign myself 
<img width="467" alt="Screen Shot 2021-03-31 at 9 48 27 AM" src="https://user-images.githubusercontent.com/67783023/113181061-3f09d480-9206-11eb-9df5-5edac9bbb6bc.png">

Similarly I cannot edit the template, i think it is because I do not have necessary rights.

**Comment by gentzkow** [comment]
Looks like you weren't added to that repo as a collaborator. I just sent
you an invite.

>

**Comment by Houdanait** [comment]
Thanks  ! @gentzkow

**Comment by Houdanait** [comment]
**Summary of this issue**

The goal of this issue was to change the indicated syntax for brew download in the ReadMe as described [here](https://github.com/gentzkow/template/issues/36#issue-845253511). 
Change of syntax was implemented in commit [e36f0de0e](https://github.com/gentzkow/template/pull/37/commits/e36f0de0e93f19efda8c330a6840b4c7db6dc645) and merged to master in [2f04d899c6](https://github.com/gentzkow/template/commit/2f04d899c6565608540a18bb71c07ba5406af5cd) via PR #37
